### ISSUE TITLE
Ensure we have styled focus states 

### ISSUE THREAD

**Issue by ststimac** [webhintio/webhint.io#454]
While tabbing through the documentation page to test something else, I noticed the subheadings that have element id tags for linking to that specific section are being focused on but there are no focus styles for those headings or the link button next to them. 

Need to add focus styles for those & do an audit of other pages on the site to make sure everything that needs a focus state has the appropriate styling.
